src/components/molecules/BackgroundBreadcrumb/BackgroundBreadcrumb.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    class="bg-cover bg-center h-[22rem] flex items-center justify-center text-center"
```

Comment:
![image](https://github.com/quyen46919/Rafcart_fe/assets/69032266/1a02236e-52ea-4ba7-a7ed-0a505347b0e2)
trường hợp ảnh siêu to thì chưa giới hạn width - height
text title dài thì breadcrumb bị lệch